### ISSUE TITLE
icons with questionable copyright in notmyidea theme

### ISSUE THREAD

**Issue by josch** [getpelican/pelican#2179]
Pelican distributes several icons in pelican/themes/notmyidea/static/images/icons/ which do not look as if the pelican authors would have permission to distribute them. What is the copyright on those files?

**Comment by jonaslb** [comment]
IANAL but i think it's ["nominative use"](https://en.wikipedia.org/wiki/Nominative_use), ie. the equivalent of fair use for trademarks.

**Comment by josch** [comment]
I'm not a lawyer either but "fair use" is an US concept and does not apply elsewhere in the world. So Debian will definitely have to remove these files.

github is a US company so it's probably fine for you to continue distributing these files. But you would make it easier for other distributions like Debian if you would provide a replacement for these icons out of the box.

If you don't care, just close this bug report and no hard feelings. We will just remove the icons in Debian. :)

**Comment by jonaslb** [comment]
I want to add that I don't represent the Pelican team, if you were in doubt :) Just thought I had some usable insight!

**Comment by jorgesumle** [comment]
I don't think that a default theme should have any third-party icons, the fact that they are copyrighted is even worse. That's very biased. It would be better to be neutral. Also, I don't think that the icons are used.

> If you don't care, just close this bug report and no hard feelings. We will just remove the icons in Debian. :)
 
@getpelican/reviewers  what do you think?

**Comment by josch** [comment]
It does not make sense to close bugs like this. Either this is a bug and then it will remain open until it's fixed. Or it's not a bug and then you can close it immediately.

**Comment by josch** [comment]
I'm not a lawyer either but Debian declared the icons as non-free and removed the theme from its package. See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=868047

**Comment by avaris** [comment]
So, which icons are copyrighted? And where can one find that information?

**Comment by josch** [comment]
Ideally, the copyright information of each piece of the these should've been supplied to you by the theme author and then be recorded somewhere. This is to make sure that when you distribute pelican, its users can use it under the same terms that you chose for pelican itself.

Unfortunately we now found out only after the fact that some of the material contained in this theme cannot fall under the AGPL as the rest of the project. Here is a small summary about social media icons: https://www.logodesignteam.com/blog/copyright-and-logo-usage-of-social-media-logos-and-icons/ Especially the "don't"'s make it clear that the icon is not free software any more.

I think the first step is to identify all the creative work that cannot possibly be licensed AGPL like the rest of the project. After that, I think you have two options:

 1. remove or replace the affected material so that the project as a whole can be redistributed under the terms of the AGPL again
 2. make a precise list somewhere in your project where you list all content that is not covered by the same license as the rest of pelican and note down what the terms of use actually are

Both ways solve the problem that a user downloading pelican will think: "Hey, everything I have here falls under the terms of the AGPL so I can do what the AGPL allows me to do with everything in this git repository, right?" Unfortunately this is not the case with the icons and because their current license is rather a "all-rights-reserved" type, the whole theme got removed in Debian.

**Comment by avaris** [comment]
Well, the icons are mostly a "nice-to-have" rather than "must-have", so removing them wouldn't be the end of the world. We could extend the `SOCIAL` config so that the user can provide the icon as well.

I'm not sure replacing them (depending on "with what") is a valid option though, since they specifically state "do not manipulate". Unless we put something generic there.

2nd option is also appealing. Would something like this comply?

> Brand icons provided in `notmyidea` theme are for convenience and they are trademarks of respective entities. Refer to the their brand guidelines for their usage.

**Comment by ryentzer** [comment]
I read the article that @josch sent and I have a basic understanding of this as a designer. I think we could and should remove any icon that is not represented in a way that the company allows for. Many of these social sites have a brand resource that describes what can and can't be done with the logo. Perhaps, we could replace the incorrect ones with a corrected version, or if unable to find a resource for using the icon, remove it altogether?

**Comment by justinmayer** [comment]
@avaris was kind enough to volunteer and remove the relevant icons from the current default Pelican theme, so this topic can now be put to rest. Thanks to everyone who contributed to the discussion of this issue.
